sdk/src/main/java/com/mapbox/maps/MapProvider.kt

```diff
@@ -34,5 +40,43 @@ internal object MapProvider {
     telemetry: MapTelemetry
   ) = MapPluginRegistry(MapDelegateProviderImpl(mapboxMap, mapController, telemetry))
 
+  fun getMapTelemetryInstance(context: Context, accessToken: String): MapTelemetry {
+    if (!::mapTelemetry.isInitialized) {
+      mapTelemetry = MapboxModuleProvider.createModule(MapboxModuleType.MapTelemetry) {
+        paramsProvider(context, accessToken, MapboxModuleType.MapTelemetry)
+      }
+    }
+    mapTelemetry.onAppUserTurnstileEvent()
+    return mapTelemetry
+  }
+
+  /**
+   * Provides parameters for Mapbox default modules, recursively if a module depends on other Mapbox modules.
+   */
+  private fun paramsProvider(
```

Comment:
Not sure why other module types are listed since the only usage is with `MapboxModuleType.MapTelemetry` module type, maybe just inline the parameters? Up to you